ui/components/institutional/jwt-url-form/jwt-url-form.js

```diff
@@ -8,8 +8,7 @@ import {
   FlexDirection,
   TextVariant,
 } from '../../../helpers/constants/design-system';
-import { Box, Button } from '../../component-library';
-import { Text } from '../../component-library/text/deprecated';
+import { Box, Button, Text } from '../../component-library';
 import JwtDropdown from '../jwt-dropdown';
 
 const JwtUrlForm = (props) => {
```

Comment:
Small update to secondary button to use correct component API

<img width="721" alt="Screenshot 2023-07-17 at 9 43 03 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/2a77b1bf-f49a-4779-b90d-67d5e9e3a856">
<img width="1430" alt="Screenshot 2023-07-17 at 9 44 21 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/44936807-ebf8-4c30-94fa-19fa495bbf60">